### PR TITLE
Adding alerting for 5XX status codes from application and LB

### PR DESCRIPTION
**Pull request by laurence-bird**
Same as in the profiles service. I'll run the cloudformation once this is merged to master

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -218,3 +218,39 @@ Resources:
...
+        Dimensions:
+          - Name: LoadBalancer
+            Value: !GetAtt 'LoadBalancer.LoadBalancerFullName'
+        MetricName: HTTPCode_Target_5XX_Count
+        AlarmActions:
+          - !ImportValue "CloudwatchPagerDutyTopic"
+        OKActions:
+          - !ImportValue "CloudwatchPagerDutyTopic"
+        Period: 60
+        EvaluationPeriods: 3
```
Comment (cb372) [inline cloudformation.yaml]: I wonder if this is a bit too lenient? It means there must be at least 1 error per minute for 3 consecutive minutes before the alarm triggers. For all of our load balancers (especially profiles service) we probably want to be as sensitive as possible and trigger an alarm if we see a single 5xx error. (If that turns out to be noisy we can tweak it later.)

File: cloudformation.yaml [hunk 2]
```diff
@@ -218,3 +218,39 @@ Resources:
...
+        Statistic: Sum
+        Threshold: 0
```
Comment (cb372) [inline cloudformation.yaml]: Shouldn't the threshold be 1, as we are using `GreaterThanOrEqualToThreshold`?

File: cloudformation.yaml (hunk 1 above)
Comment (laurence-bird) [inline cloudformation.yaml]: Agreed, have updated for this and the other PRS outstanding

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 91](https://circleci.com/gh/ovotech/comms-template-manager/91) at 2017-03-23 16:48 GMT.

**Pull request merged**